package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-grants-eoi",
-  "version": "1.9.1",
+  "version": "1.9.2",
```

Comment:
Should probably change the version to 1.10.0

We use the semantic versioning scheme, so the numbers related to MAJOR.MINOR.PATCH

As you are adding new pages, I would say this is a minor change so 1.10.0 makes more sense than 1.9.2 which is a patch (bug fix)